packages/grafana-toolkit/package.json

```diff
@@ -102,11 +102,11 @@
     "simple-git": "^1.132.0",
     "style-loader": "^0.23.1",
     "terser-webpack-plugin": "^2.3.4",
-    "ts-jest": "24.1.0",
+    "ts-jest": "26.3.0",
     "ts-loader": "6.2.1",
-    "ts-node": "8.8.1",
-    "tslib": "1.10.0",
-    "typescript": "3.9.3",
+    "ts-node": "9.0.0",
+    "tslib": "2.0.1",
+    "typescript": "4.0.2",
```

Comment:
This change will require testing sample plugins. This may come handy https://github.com/grafana/grafana/blob/master/packages/README.md#setting-up-grafana-packages-for-local-development